app/controllers/users_controller.rb

```diff
@@ -1097,6 +1098,19 @@ def revoke_account
     end
   end
 
+  def revoke_auth_token
+    user = fetch_user_from_params
+    guardian.ensure_can_edit!(user)
+
+    UserAuthToken.where(user_id: user.id).destroy_all
```

Comment:
Because we're not checking for return value by `destroy_all` code here. You probably want to do 

```
UserAuthToken.where(user_id: user.id).each(&:destroy!)
```

Reply:
Noted. Thanks!